### PR TITLE
German translation

### PR DESCRIPTION
**Pull request by mKay00**
... and corrected a typo in english template.json file

Issue #5

### CODE REVIEW

File: locales/de-DE/about.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+### Warum PoGo-UWP?
+ - Weil es immer cool ist, neues zu lernen
+ - Weil es möglich ist
+ 
+### Momentane Features
+ - Login mit Pokemon™ Trainer Club Accounts
+ - Gleiche Benutzeroberfläche wie die Android/iOS Version, aber nur 2D
+ - Benutze die Karte um sowohl Pokémons als auch PokéStops in deiner Nähe zu sehen
+ - Fange Pokémons
```
Comment (FloMoTionGo) [inline locales/de-DE/about.md]: o_O Plural of Pokemon is still Pokemon ;)

File: locales/de-DE/about.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+PoGo-UWP ist ein UWP (Universal Windows Platform) Client für Niantics Pokémon™ Go Android/iOS Spiel. Ein Client erlaubt dir, mit deinen Freunden auf Android oder iOS Geräten in der selben Spielwelt zu spielen.
+
+### Warum PoGo-UWP?
+ - Weil es immer cool ist, neues zu lernen
+ - Weil es möglich ist
+ 
+### Momentane Features
+ - Login mit Pokemon™ Trainer Club Accounts
+ - Gleiche Benutzeroberfläche wie die Android/iOS Version, aber nur 2D
+ - Benutze die Karte um sowohl Pokémons als auch PokéStops in deiner Nähe zu sehen
```
Comment (FloMoTionGo) [inline locales/de-DE/about.md]: give this sentence a comma or two ... at least before "als"

File: locales/de-DE/contribute.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+## Unterstützen
+
+Um Feedback zu geben und/oder Probleme zu melden, benutze bitte [GitHub Issues](https://github.com/ST-Apps/PoGo-UWP/issues) (du brauchst einen kostenlosen GitHub Abbount).
```
Comment (FloMoTionGo) [inline locales/de-DE/contribute.md]: "Abbount" --> Account

File: locales/de-DE/about.md (hunk 1 above)
Comment (mKay00) [inline locales/de-DE/about.md]: Oha, didn't know that :D
And thanks for all other corrections ;)

**Comment by FloMoTionGo** [comment]
I dont have experience in opening a pull request, thus I post the pre-final german translation here.
Merged your stuff with mine ... Further improvements appreciated ;)

[de-DE.zip](https://github.com/PoGo-Devs/PoGo-Devs.github.io/files/426940/de-DE.zip)

forgot to change: "tille-loading" in template.json

**Comment by mKay00** [comment]
Okay, everything updated.
It's now complete.

**Comment by FloMoTionGo** [comment]
was just about to make my own PR, but thx anyway. Now I know how "difficult" it is to make a PR ;)

File: locales/de-DE/template.json [hunk 4]
```diff
@@ -1,20 +1,20 @@
...
 	"title-telegram": "Telegram",
-	"title-loading": "Loading...",
-	"title-newest": "Newest!",
-	"title-show-log": "Show change log",
-	"title-hide-log": "Hide change log",
-	"title-show-downloads": "Show Downloads",
-	"title-hide-downloads": "Hide Downloads"
-}
\ No newline at end of file
+	"tille-loading": "Wird geladen ...",
```
Comment (FloMoTionGo) [inline locales/de-DE/template.json]: öhhh, sry vergessen zu ändern: tille -> title

**Comment by mKay00** [comment]
Yeah, I am struggling a bit with GitHub as well :D

**Pull request merged**